### PR TITLE
Speed up tests (swc, plotting)

### LINKED ISSUE #468: Speed up plotting (reduce mesh resolution)
The level of detail of the meshes (create_cone_frustum_mesh, create_cylinder_mesh, create_sphere_mesh) is the main bottleneck for plotting faster (for morph plots).
However, all three of this methods start with a line defining a 'resolution'. I think it would be nice to make this a parameter and allow to produce low-resolution but fast plots.
Also useful to have the plot tests run faster ;)

### PR DESCRIPTION
**Pull request by fabioseel**
Main contributions:
- swc tests now significantly faster (change to sparse solver)
- plotting tests a lot faster
  - exposed 'resolution' of mesh generation (no proper exposition to user (yet)) fixes #468
  - splitted plot volume test for more granular analysis

minor contributions:
- added test fixtures for compartment, cell, morph_cell, branch and network -> not sure how much it helps / improves readability here. Could also revert these changes. (might be a first step for #449)

### CODE REVIEW

File: tests/test_plotting_api.py [hunk 1]
```diff
@@ -19,31 +19,58 @@
 from jaxley.synapses import IonotropicSynapse
 
 
-def test_cell():
-    dirname = os.path.dirname(__file__)
-    fname = os.path.join(dirname, "swc_files", "morph.swc")
-    cell = jx.read_swc(fname, nseg=4)
+@pytest.fixture(scope="module")
```
Comment (jnsbck) [inline tests/test_plotting_api.py:22]: This is awesome! I think we could even move them to a separate file, to make them available to all other tests. `tests/fixtures.py`.

Comment (jnsbck) [reply tests/test_plotting_api.py:22]: Feel free to replace as many of the other tests that rely on `comp` or `net` etc. with these fixtures. But also fine if you don't and then just leave #449 open and just add a quick comment to it.

**Review by jnsbck — APPROVED** [review]
Not a lot to critique here. Left only a couple comments. Feel free to merge this once adressed. :)

**Review by michaeldeistler — APPROVED** [review]
Awesome! Good to go also from my side! @fabioseel let us know if/when we can merge it! Thanks a ton!

**Comment by jnsbck** [comment]
Awesome, thanks a ton!

**Pull request merged**